.github/workflows/test.yml

```diff
@@ -73,7 +73,9 @@ jobs:
 
       - name: Install integration test dependencies
         id: minikube
-        uses: medyagh/setup-minikube@5a9a7104d7322fa40424de8855c84685e89cefd7
+        uses: medyagh/setup-minikube@master
```

Comment:
This means that there can be other kinds of sudden failures due to constantly changing minikube versions?

Reply:
Kubernetes only does updates about 3 times a year, so I doubt that that will happen often, but yes that is a concern.